packages/server/src/lib/utils.js

```diff
@@ -0,0 +1,86 @@
+const assert = require('assert');
+const _ = require('lodash');
+
+/**
+ * @param className
+ * @param options
+ * @param list
+ * @return {string[]}
+ */
+module.exports.diff = function diff(className, options, list) {
+  const difference = _.difference(_.keys(options), list);
+  if (difference.length > 0) {
+    console.warn(`${className} don't expect "${difference.join(', ')}".`);
```

Comment:
It should probably be ``doesn't``, not ``don't``

Reply:
Yes, sorry